core/src/config/sequelize.ts

```diff
@@ -97,12 +97,11 @@ export const DEFAULT = {
       migrations: [join(__dirname, "..", "migrations")],
       storage, // only used for sqlite
       pool: {
-        max:
-          dialect === "sqlite"
```

Comment:
lose this sqlite affordance on purpose?

Reply:
Good catch! We already have a check that ensures you only have 1 worker for SQLite, but the `+1` gets you in trouble...